Default.sublime-keymap

```diff
@@ -113,89 +113,79 @@
         "command": "gs_status_open_file",
         "context": [
             { "key": "setting.command_mode", "operator": "equal", "operand": false },
-            { "key": "setting.git_savvy.status_view", "operator": "equal", "operand": true },
-            { "key": "selector", "operator": "equal", "operand": "git-savvy.status meta.git-savvy.status.section meta.git-savvy.status.file" }
```

Comment:
We remove all "selector" constraints from the keymap because they were not sufficient anyway.